script/diaspora-dev

```diff
@@ -158,6 +158,15 @@ print_usage_full() {
 
 # ----- Helper functions -----
 
+dia_docker_compose() {
```

Comment:
Would be nice to keep consistent with the rest of the functions and add a little descriptive comment to the function.